src/System.IO.FileSystem/tests/Performance/Perf.Directory.cs

```diff
@@ -63,7 +63,7 @@ public string GetTestDeepFilePath(int depth)
             StringBuilder sb = new StringBuilder(5000);
             for (int i = 0; i < depth; i++)
             {
-                sb.Append(@"\a");
+                sb.Append(Path.DirectorySeparatorChar + "a");
```

Comment:
Not sure if this will optimize away. You might want to make a temporary `string directory = Path.DirectorySeparatorChar + "a"` outside of the loop to be safe.

Reply:
It is not getting optimised away. should i still change it ?

Reply:
Yes, that was my point. Do it to be safe- if you _know_ it isn't of course do it. No point in putting a bunch of unneeded stress on when it is such a simple change.